packages/p2p-v2/src/pages/buy-sell/components/SortDropdown/SortDropdown.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { TSortByValues } from '@/utils';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Button, Dropdown, useDevice } from '@deriv-com/ui';
+import SortIcon from '../../../../public/ic-cashier-sort.svg';
+
+type TSortDropdownProps = {
+    list: { text: string; value: string }[];
+    onSelect: (value: TSortByValues) => void;
+    setIsFilterModalOpen: (value: boolean) => void;
+    value: TSortByValues;
+};
+
+const SortDropdown = ({ list, onSelect, setIsFilterModalOpen, value }: TSortDropdownProps) => {
+    const { isMobile } = useDevice();
+
+    if (isMobile) {
+        return (
+            <Button
+                className='w-[3.2rem]'
+                icon={<SortIcon className='absolute' data-testid='dt_p2p_v2_sort_dropdown_button' />}
+                onClick={() => setIsFilterModalOpen(true)}
+                variant='outlined'
+            />
+        );
+    }
+
+    return (
+        <Dropdown
+            dropdownIcon={<LabelPairedChevronDownMdRegularIcon />}
+            label='Sort by'
+            list={list}
+            name='hi'
+            onSelect={(value: string) => onSelect(value as TSortByValues)}
+            value={value}
+        />
+    );
+};
+
+export default SortDropdown;
```

Comment:
can we include the spec file as well?

Reply:
its already covered by BuySellTable.spec but i'll add it in the description
![Screenshot 2024-02-14 at 4 43 31 PM](https://github.com/binary-com/deriv-app/assets/103412909/b5ad6fac-8d68-4fb0-b0ae-779af38d3b99)

Reply:
same with the buyselltablerow its covered by BuySellTable

Reply:
![Uploading Screenshot 2024-02-14 at 4.48.03 PM.png…]()